plugins/pencilblue/controllers/actions/admin/sites/new_site.js

```diff
@@ -0,0 +1,82 @@
+/*
+ Copyright (C) 2015  PencilBlue, LLC
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+module.exports = function NewSiteActionModule(pb) {
```

Comment:
Route handler functions can be specified as part of the route definition.  If you specify the handler then you can group common actions such as "edit site" and "new site" in the same file and reduce the amount of code that needs to be written and maintained.